scripts/haml.rb

```diff
@@ -1,6 +1,15 @@
 require 'rubygems'
 require 'bundler/setup'
 require 'haml'
+require 'rails_i18n'
```

Comment:
Noting here that I'm including `rails_i18n` but in practical terms I think we are only really relying on the underlying `i18n` gems behaviour as we don't trigger any of the railties code in this context (as far as I know!). I still think this makes sense though as it's ultimately what will get used on the rails end.